scripts/tests/provisioning/start-vm-using-expect.sh

```diff
@@ -26,17 +26,18 @@ usage() {
 	cat <<EOF
 $(basename $BASH_SOURCE) -h|--help
   Print this usage information and exit.
-$(basename $BASH_SOURCE) [-i ipk_feed_dir] nilrt_vm_path
+$(basename $BASH_SOURCE) [-i ipk_feed_dir] nilrt_vm_path expect_file
+                         [expect_file ...]
   Start the NILRT VM, setup the feeds  and run the expect file
 
 Options:
--i|--ipk-feed  Path to a local IPK feed, which will be added to the VM's opkg
-               configuration.
+  -i|--ipk-feed  Path to a local IPK feed, which will be added to the VM's opkg
+                 configuration.
 
 Arguments:
-nilrt_vm_path  Path to an already-provisioned NILRT VM directory. (See the
-               buildVM.sh script in the NILRT.git source.)
-expect_file    Path to the .expect file
+  expect_file    Path to the .expect file
```

Comment:
This should be expect_files and "Path to the .expect files" now?

Reply:
I don't think it should be plural, no. Each shlex word is a singular file path, so I think the help-text should reflect that.